content/includes/breadcrumbs.html

```diff
@@ -8,13 +8,7 @@
       {% assign current_page = breadcrumb_path | last %}
       {% for crumb in breadcrumb_path %}
         {% if current_page.name == crumb.name %}
```

Comment:
Since we only care about the `else` here, maybe switch this to `{% if current_page.name != crumb.name %}` to simplify